issue_template.md

```diff
@@ -0,0 +1,20 @@
+### Your system information
+
+* Steam client version:
+* Distro / OS version (e.g. Arch Linux 64-bit): 
+* CPU vendor and model (e.g. Intel i7-7700HQ):
+* Graphics card/chip vendor and model (e.g. NVIDIA GTX 1060): 
+* Graphics driver version (e.g. Nvidia 396.55): 
```

Comment:
Well, my comment wasn't meant as blocker but as an info. A Mesa version would cover all GPUs and Mesa provides the standard, out of the box drivers for all graphics APIs that are common for open source software. Installing proprietary drivers is an option but not the default. And it is never something we like to see on Linux, because beside intellectual property which can be protected by putting small bits of code in firmware blobs, there seem to be no valid reasons for someone who sells __hardware__ to hide everything other than shady and suspicious ones. 

By adding the LLVM version in the example you also remind people who have a non standard Mesa version to tell which compiler was used to assign issues better to projects. But I would suggest to mark this entry as optional because when you install Ubuntu 18.04 as an example and don't fiddle, it's clear how the driver has been built and you shouldn't be bothered with LLVM and stuff that might confuse you.

When people use the proprietary drivers it should __not__ be necessary to show them how their version string looks like - everyone who is able to type numbers should know how a version of a program looks like. The danger that someone misses to tell anything relevant does only exist when the driver can be compiled by third parties because it's open source.